maven-archetype-plugin/src/main/java/org/apache/maven/archetype/ui/generation/DefaultArchetypeGenerationConfigurator.java

```diff
@@ -313,46 +311,25 @@ else if ( !archetypeGenerationQueryer.confirmConfiguration( archetypeConfigurati
         request.setProperties( properties );
     }
 
-    private String getTransitiveDefaultValue( String defaultValue, ArchetypeConfiguration archetypeConfiguration,
-                                              String requiredProperty, Context context )
+    private String getTransitiveDefaultValue( String defaultValue, String requiredProperty, Context context )
```

Comment:
This method name is unclear. `evaluateTemplate` would be clearer.

Reply:
looks like this could be static since it doesn't depend on instance state (unless I'm missing something)